client/landing/gutenboarding/store/reducer.ts

```diff
@@ -11,7 +11,7 @@ import { ActionType, SiteType, Vertical } from './types';
 import * as Actions from './actions';
 
 const siteType: Reducer< SiteType, ReturnType< typeof Actions[ 'setSiteType' ] > > = (
-	state = SiteType.BLOG,
+	state = '',
```

Comment:
I don't think that the empty string is currently a valid value for a `SiteType`; we might have to change [its definition](https://github.com/Automattic/wp-calypso/blob/bce08745a805e3ecdd4bfaef8eb6e402aaea5531/client/landing/gutenboarding/store/types.ts#L9-L13) to allow for that. (Maybe `null` would be a better way to represent empty state here; but with TypeScript, that might arguably be an implementation detail.)